docker-compose.test.yml

```diff
@@ -30,8 +30,8 @@ services:
       service: base
     environment:
       - APP_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://inspirehep:dbpass123@test-database:5432/inspirehep
-      - APP_BROKER_URL=amqp://guest:guest@test-rabbitmq:5672//
-      - APP_CELERY_RESULT_BACKEND=amqp://guest:guest@test-rabbitmq:5672//
+      - APP_CELERY_BROKER_URL=pyamqp://guest:guest@test-rabbitmq:5672//
```

Comment:
is this really needed? `amqp` is still in the installation docs: http://docs.celeryproject.org/en/latest/getting-started/brokers/rabbitmq.html#installation-configuration

Reply:
> The old legacy “amqp” **result** backend has been deprecated, and will be removed in Celery 5.0.

(emphasis mine), I am talking about the broker, not the result backend.

Reply:
Ah, I missed that! Then I don't know the answer off the top of my head. *Probably* I did it because of some deprecation notice or something like that. Let me revert that change and see what happens.